synapse/lib/view.py

```diff
@@ -227,13 +236,36 @@ def pack(self):
 
         await synt.worker(runStorm())
 
+        necount = 0
+        editformat = opts.get('editformat', 'nodeedits')
+
         while True:
 
             mesg = await chan.get()
+            kind = mesg[0]
+
+            if kind == 'node:edits':
+                if editformat == 'nodeedits':
+                    yield mesg
+                    continue
+
+                if editformat == 'none':
+                    continue
+
+                if editformat == 'count':
+                    necount += 1
```

Comment:
maybe count should be the count of the total node edits to reflect the work the system is doing?

Reply:
How would that be different from what we're currently doing?

Reply:
I think it means increment by the number of edits in the nodeedits

Reply:
Yep!  The total number of edits within the nodeedits.